core/client/fs/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -100,4 +101,8 @@ public int read(ByteBuffer byteBuffer, int off, int len) throws IOException {
     }
     return nread;
   }
+
+  @Override
+  public void unbuffer() {
```

Comment:
If the default implementation is empty, will other streams be affected?

Reply:
Will not affect